pkg/importer/hcl2.go

```diff
@@ -288,6 +292,46 @@ func makeResourceOptions(state *resource.State, names NameTable, addedRefs map[s
 			Value:  cty.True,
 		})
 	}
+	if state.RetainOnDelete {
+		v := cty.BoolVal(state.RetainOnDelete)
+		resourceOptions = appendResourceOption(resourceOptions, "retainOnDelete", &model.LiteralValueExpression{
+			Tokens: syntax.NewLiteralValueTokens(v),
+			Value:  v,
+		})
+	}
+	if len(state.IgnoreChanges) > 0 {
+		ignoreChanges := make([]model.Expression, len(state.IgnoreChanges))
+		for i, prop := range state.IgnoreChanges {
+			v := cty.StringVal(prop)
+			ignoreChanges[i] = &model.LiteralValueExpression{
+				Tokens: syntax.NewLiteralValueTokens(v),
+				Value:  v,
+			}
+		}
+		resourceOptions = appendResourceOption(resourceOptions, "ignoreChanges", &model.TupleConsExpression{
+			Tokens:      syntax.NewTupleConsTokens(len(ignoreChanges)),
+			Expressions: ignoreChanges,
+		})
+	}
+	if state.DeletedWith != "" {
+		name, ok := names[state.DeletedWith]
+		if !ok {
+			return nil, fmt.Errorf("no name for deletedWith %v", state.DeletedWith)
+		}
+		resourceOptions = appendResourceOption(resourceOptions, "deletedWith", newVariableReference(name))
+	}
+	if state.ImportID != "" {
+		v := cty.StringVal(state.ImportID.String())
+		resourceOptions = appendResourceOption(
+			resourceOptions,
+			"importID",
```

Comment:
This would need to change to `"import"` too I guess if we followed my suggestion above.